src/components/share-structure/EditShareStructure.vue

```diff
@@ -240,11 +240,11 @@ export default class EditShareStructure extends Mixins(CurrencyLookupMixin) {
         (v: string) => !(this.shareClasses
           .find((s, index) => {
             // Don't apply uniqueness check to self
-            return index !== this.activeIndex && s.name.split(' Shares')[0].toLowerCase() === v.toLowerCase()
+            return index !== this.activeIndex && s.name?.split(' Shares')[0].toLowerCase() === v.toLowerCase()
```

Comment:
Not the happiest about these for fixes, but I wanted to get the unit tests to pass - if we need to fix these another way, we can create a ticket and merge with the rest of the tests passing.